source/mpi_shared_solid_solver.cpp

```diff
@@ -162,6 +162,14 @@ namespace Solid
           spacedim,
           PETScWrappers::MPI::Vector(locally_owned_scalar_dofs,
                                      mpi_communicator)));
+
+      cellwise_stress = std::vector<Vector<double>>(
+        6, Vector<double>(triangulation.n_active_cells()));
+
+      for (unsigned int i = 0; i < cellwise_stress.size(); ++i)
+        {
+          cellwise_stress[i].reinit(triangulation.n_active_cells());
```

Comment:
If ``reinit`` is called here anyway, just use the default constructor at line 167.